lib/tasks/import.rake

```diff
@@ -28,4 +28,47 @@ namespace :import do
     puts "Missing ETDs: #{missing_etds.inspect}" if missing_etds.present?
     puts "Import Complete #{Time.zone.now}"
   end
+
+  task :adrl_arks, [:file_path] => :environment do |_t, args|
+    puts "Beginning import #{Time.zone.now}"
+    # CSV file contents would be
+    # merritt,proquest
+    # ark:/13030/m00999g9,ProQuestID:0035D
+    csv = CSV.read(args[:file_path], headers: true)
+    count = 0
+    missing_files = non_adrl = []
+
+    # rewrite CSV with each run
+    # using 'w' mode
+    CSV.open("tmp/adrl_arks.csv",
+             "w",
+             write_headers: true,
+             headers: %w[proquest merritt ucsb]) do |row|
+      ETD.find_each do |etd|
+        next if etd.merritt_id.present?
+
+        pq_name = etd.file_sets.first.original_file.file_name.first
+        if pq_name.blank?
+          missing_files << etd.id
+        else
+          pq_id = "ProQuestID:" + pq_name.split("_").last.split(".").first
+          match = csv.find { |r| r["proquest"].strip == pq_id }
+          if match.present?
+            row << [
+              match["proquest"].strip,
+              match["merritt"].strip,
+              etd.identifier.first
+            ]
+            count += 1
+          else
+            non_adrl << match["merritt"]
+          end
+        end
+      end
+    end
+    puts "Merritt ETDs not found in ADRL #{non_adrl}"
```

Comment:
When running this task in staging, is the output something like:

```
Beginning import {timestamp}
Merritt ETDs not found in ADRL {some longish list of arks}
No files found for ETDs: []
Imported 62 ETD Arks
Import Complete {timestamp}
```

Just checking that i'm understanding the expected output.